tsconfig.typedoc.json

```diff
@@ -71,8 +69,11 @@
     "theme": "default",
     "includeVersion": true,
     "hideGenerator": true,
-    "listInvalidSymbolLinks": true,
     "excludePrivate": true,
-    "includes": "packages"
+    "includes": "packages",
+    "treatWarningsAsErrors": false,
+    "validation": {
+      "invalidLink": true
+    }
```

Comment:
[q] Could you give some context why "treatWarnignsAsErrors" is false?

Reply:
Because otherwise Github actions will fail right now, it will be set to true after the 2ns PR

Reply:
While fixing the links I already set it to true